OpenMage/magento-lts
# Added possibility to set backend locale per every admin user

This PR adds the possibility to set a `locale` for the `backend` for every `backend user`, adding a column to the `admin_user` table and the UI components to use this new column.

This is the new field added to the "System -> My Account" form:

<img width="594" alt="Screenshot 2023-03-14 alle 22 39 50" src="https://user-images.githubusercontent.com/909743/225158469-968b020f-2b51-44fa-aabf-24454c3d88c9.png">


### Manual testing scenarios (*)
1. login to the backend
2. click "system -> my account"
3. set the backend locale using the new field
4. save
5. logout
6. login to the backend again
7. the locale of the backend should be the one selected
